README.md

```diff
@@ -2,7 +2,9 @@
 
 This repository contains the vault for the Obsidian Developer Docs.
 
-A hosted version of this vault is available at https://docs.obsidian.md/.
+A more accessible version of this vault is available at https://docs.obsidian.md/. (recommended for new contributors) 
```

Comment:
The "recommended for new contributors" part is confusing, because you can't contribute to this repo from the docs page.
Also, I would not use "A more accessible version", because it's also a bit confusing.